server/commands/cycle.js

```diff
@@ -1,47 +1,64 @@
 /* global graphQLFetcher, notifyUser, commandsConfig */
 
-function invokeLaunchCycleAPI(lgJWT) {
+function invokeUpdateCycleStateAPI(state, lgJWT) {
   const baseURL = process.env.NODE_ENV === 'development' ? 'http://game.learnersguild.dev' : 'https://game.learnersguild.org'
   const mutation = {
-    query: 'mutation { launchCycle { id } }'
+    query: 'mutation($state: String!) { updateCycleState(state: $state) { id } }',
+    variables: {state},
   }
   return graphQLFetcher(lgJWT, baseURL)(mutation)
     .then(data => data.launchCycle)
 }
 
-function launchCycle(commandInfo) {
+function handleUpdateCycleStateCommand(commandInfo, state, msg) {
   try {
     const {lgJWT, lgPlayer} = Meteor.user().services.lgSSO
     if (!lgJWT || !lgPlayer) {
       throw new Error('You are not a player in the game.')
     }
-    notifyUser(commandInfo.rid, '🚀  Initiating Launch... stand by.')
-    invokeLaunchCycleAPI(lgJWT)
-      // unless our API invocation fails, we'll stay quiet because the server will notify
-      // the user via the web socket
-      // .then(response => console.log(`[LG SLASH COMMANDS] '/cycle launch' API response: ${response}`))
+    notifyUser(commandInfo.rid, msg)
+    invokeUpdateCycleStateAPI(state, lgJWT)
       .catch(error => {
         console.error(error.stack)
         RavenLogger.log(error)
-        notifyUser(commandInfo.rid, '**FATAL**: API invocation failed.')
+        notifyUser(commandInfo.rid, `**ERROR**: ${error.message}`)
       })
   } catch (errorMessage) {
     notifyUser(commandInfo.rid, `**ERROR:** ${errorMessage.message}`)
   }
 }
 
+function showUsage(rid) {
+  notifyUser(rid, `**USAGE:**
+  \`/cycle launch\` - Form teams and launch the cycle
+  \`/cycle retro\` - Start the retrospective
+  \`/cycle help\` - Show this handy usage information
+`)
+}
+
 commandsConfig.cycle.onInvoke = (command, commandParamStr, commandInfo) => {
   const subcommands = commandParamStr.split(/\s+/).filter(subcommand => subcommand.length > 0)
   if (subcommands.length > 0) {
     const subcommand = subcommands[0]
     switch (subcommand) {
       case 'launch': {
-        launchCycle(commandInfo)
+        handleUpdateCycleStateCommand(commandInfo, 'PRACTICE', '🚀  Initiating Launch... stand by.')
+        break
+      }
+      case 'retro': {
+        handleUpdateCycleStateCommand(commandInfo, 'RETROSPECTIVE', '🤔  Initiating Retrospective... stand by.')
+        break
+      }
+      case 'help': {
+        showUsage(commandInfo.rid)
         break
       }
       default: {
         notifyUser(commandInfo.rid, '**ERROR:** Invalid action for /cycle')
+        showUsage(commandInfo.rid)
```

Comment:
@bundacia, @jeffreywescott: noticing a pattern for programs displaying both and error message and usage help when a user attempts an invalid command. It's not the end of the world obvs...but it's inconsistent with behavior I've experienced in other professional apps/programs.

My gut feeling is that it's a better UX to get just one or the other (error message or usage info) but I don't have much to back that up except for experience with other apps.

Thoughts?